### PR TITLE
fix #8: allow extension on lock screen

### LINKED ISSUE #8: 3kbSwitch Gnome Shell extension vs. Lock Screen
I found that 3kbSwitch Gnome Shell extension doesn't work when Lock Screen is active. Please, let me know if there are any ideas how to make it work.

Output when commands executed in terminal manually (3kbSwitch and the extension work fine):
```
$ id
uid=1000(cactus) gid=1000(cactus) группы=1000(cactus),4(adm),24(cdrom),27(sudo),30(dip),46(plugdev),122(lpadmin),134(lxd),135(sambashare),137(vboxusers)
$ gdbus call --session --dest org.gnome.Shell --object-path /org/g3kbswitch/G3kbSwitch --method org.g3kbswitch.G3kbSwitch.List
(true, '[{"key":"0","value":"us"},{"key":"1","value":"ru"}]')
```

Output if the same commands executed by a script during active Lock Screen (user session is still active, no logout, 3kbSwitch Gnome Shell doesn't work):
```
$ id
uid=1000(cactus) gid=1000(cactus) группы=1000(cactus),4(adm),24(cdrom),27(sudo),30(dip),46(plugdev),122(lpadmin),134(lxd),135(sambashare),137(vboxusers)
$ gdbus call --session --dest org.gnome.Shell --object-path /org/g3kbswitch/G3kbSwitch --method org.g3kbswitch.G3kbSwitch.List
Ошибка: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Объект по пути «/org/g3kbswitch/G3kbSwitch» не существует
```

Some details if necessary:
I'm trying to find a way to automatically switch keyboard layout to English in password input field of Lock Screen. By default, GNOME 42.1 (Ubuntu 22.04 LTS, X11) Lock Screen derives the keyboard layout from the last focused window.
Earlier, I could find Lock Screen activation with dbus-monitor and automatically change my keyboard layout to English (if necessary) with org.gnome.Shell.Eval() method. But this method was disabled for security reasons since GNOME 41. So, I'm trying to find another solution.

### PR DESCRIPTION
**Pull request by ntzb**
this should fix https://github.com/lyokha/g3kb-switch/issues/8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,8 @@ $ cd ../extension
 $ make install  # no sudo required!
 ```
 
+restart for changes to take effect.
```
Comment (lyokha) [inline README.md:58]: Please make this start with a capital letter, i.e. `Restart`. This should also clearly state *what* should be restarted. Did you mean `Restart Gnome Shell`?

File: extension/g3kb-switch@g3kb-switch.org/metadata.json [hunk 2]
```diff
@@ -10,6 +10,10 @@
         "44"
     ],
     "url": "https://github.com/lyokha/g3kb-switch",
-    "version" : 1
+    "version" : 1,
+    "session-modes" : [
+        "user",
+        "unlock-dialog"
+  ]
```
Comment (lyokha) [inline extension/g3kb-switch@g3kb-switch.org/metadata.json:17]: Please align this brace with other items (to keep 4-spaces indentation).

**Review by lyokha — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ntzb) [reply README.md:58]: I did a full system restart, but not sure if gnome shell restart is sufficient.
I'll change it to a full restart

File: extension/g3kb-switch@g3kb-switch.org/metadata.json (hunk 2 above)
Comment (ntzb) [reply extension/g3kb-switch@g3kb-switch.org/metadata.json:17]: maybe I'm missing something, but this has proper indentation, same as the key `shell-version`

Comment (lyokha) [reply extension/g3kb-switch@g3kb-switch.org/metadata.json:17]:
```json
    "shell-version": [
        "40",
        "41",
        "42",
        "43",
        "44"
    ],
    "url": "https://github.com/lyokha/g3kb-switch",
    "version" : 1,
    "session-modes" : [
        "user",
        "unlock-dialog"
  ]
```

to

```json
    "shell-version": [
        "40",
        "41",
        "42",
        "43",
        "44"
    ],
    "url": "https://github.com/lyokha/g3kb-switch",
    "version" : 1,
    "session-modes" : [
        "user",
        "unlock-dialog"
    ]
```

Note that now the closing brace is aligned with starting of top items and their closing braces, see that now it's on the same column with the closing bracket of `"shell-version"`.

File: README.md (hunk 1 above)
Comment (lyokha) [reply README.md]: Ok, I also do not know what should be restarted :) So, restart PC is accepted.

File: extension/g3kb-switch@g3kb-switch.org/metadata.json (hunk 2 above)
Comment (ntzb) [reply extension/g3kb-switch@g3kb-switch.org/metadata.json]: right sorry, the closing bracket.
I thought you meant the content of the array.
fixed

**Review by lyokha — APPROVED** [review]
Thanks!

**Pull request merged**